corehq/apps/es/management/commands/elastic_sync_multiplexed.py

```diff
@@ -134,6 +139,43 @@ def cancel_reindex(self, task_id):
     def reindex_status(self, task_id):
         check_task_progress(task_id, just_once=True)
 
+    def delete_index(self, cname):
+        """Deletes the older index after reindexing
+
+        This should be used during the reindex process to delete the older index.
+        The function assumes that reindex process is successful, multiplexes is turned off
+        and indexes have been swapped.
+
+        """
+        adapter = doc_adapter_from_cname(cname)
+        older_index = getattr(es_consts, f"HQ_{cname.upper()}_INDEX_NAME")
+        current_index = getattr(es_consts, f"HQ_{cname.upper()}_SECONDARY_INDEX_NAME")
+        if isinstance(adapter, ElasticMultiplexAdapter):
+            raise IndexMultiplexedException(f"""A multiplexed index cannot be deleted.
+            Make sure you have set ES_{cname.upper()}_INDEX_MULTIPLEXED to False """)
+
+        assert adapter.index_name == current_index, f"""Indexes not swapped yet.
+        Make sure after reindexing set ES_{cname.upper()}_INDEX_SWAPPED to True"""
+
+        older_doc_adapter = adapter.__class__(older_index, adapter.type)
+
+        older_index_count = older_doc_adapter.count({})
+        newer_index_count = adapter.count({})
+
+        print(f"Docs in older index - {older_index_count}")
+        print(f"Docs in newer index - {newer_index_count}")
+
+        print("\nDocs in new index should be greater than or equals to in older index")
+
+        print(f"Are you sure you want to delete the older index - {older_index}?")
+        print("WARNING: - This step can't be un-done.")
+        proceed = input("Enter y/Y to continue, any other key to cancel\n")
+        if proceed.upper() != "Y":
```

Comment:
Rather than simply entering `Y`, how about requiring a more complex input like `YES` or
```py
proceed = input(f"Type '{cname}' to delete the index: ")
if proceed == cname:
```